modules/page-memory/src/test/java/org/apache/ignite/internal/pagememory/persistence/checkpoint/IgniteConcurrentMultiPairQueueTest.java

```diff
@@ -47,16 +50,16 @@ public class IgniteConcurrentMultiPairQueueTest {
 
     private Map<Integer, Collection<Integer>> mapForCheck2;
 
+    private Integer[] arr1 = {1, 3, 5, 7, 9, 11, 13, 15, 17, 19};
```

Comment:
Interesting change, what was the motivation?

Reply:
just for beauty